test/support/test_helpers.ex

```diff
@@ -36,6 +37,27 @@ defmodule RemoteRetro.TestHelpers do
     Map.put(context, :user, user)
   end
 
+  def persist_other_user_for_retro(context) do
```

Comment:
hey @nicholalexander, i see what you mean about wanting to generalize user creation logic. given that each test helper takes and returns a context, maybe we could simply change the existing `persist_user_for_retro` to `persist_users_for_retro`, and have the callsites define `@users` as a list of users. That way, we can keep the logic in the existing (though altered) method and avoid introducing a new method just for this 'other user' case

Reply:
cool.